modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,128 +8,262 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies", title="Runtime Fabric"]
 
-The load balancer configuration is defined by a list of mapping rules that describe how an input URL should be translated into calls to different CloudHub apps.
-Mapping rules are attributes of the load balancer's SSL endpoint.
-When you create a mapping rule, you need to specify a certificate CN. Omitting the `[certificateName]` parameter adds the mappings to the default endpoint.
 
-When you create a simple matching rule, one input address is matched to the defined output: the endpoint of one of your applications.
-Instead of using literal matchings, you can also use a pattern to match a variable, such as input text to an endpoint.
+The CloudHub dedicated load balancer (DLB) routes requests from clients to Mule apps deployed within the VPC.
+_Mapping rules_ enable you to rename requests to the DLB (input URL) to a different Mule application name and domain.
 
-By using proxy rules, you can map a domain or subdomain to one of your Mule applications running in CloudHub.
+You can either define mapping rules when you create the DLB or add them to an existing DLB using Runtime Manager, the command-line interface (CLI), or the CloudHub API.
 
-== Using Patterns in Mapping Rules
+When you create a simple matching rule, one input address matches the defined output: the endpoint an application.
+You can also use a pattern to match a variable, such as input text to an endpoint.
```

Comment:
This is from the existing doc:
> When you create a simple matching rule, one input address is matched to the defined output: the endpoint of one of your applications. Instead of using literal matchings, you can also use a pattern to match a variable, such as input text to an endpoint.

Reply:
Added back "Instead of using literal matchings," 

This is the intro - I don't think we want to talk about the details and limitations of pattern matching here.